cordova-lib/src/cordova/restore-util.js

```diff
@@ -63,7 +64,27 @@ function installPlatformsFromConfigXML(platforms, opts) {
             // Combining arrays and checking duplicates.
             comboArray = pkgJsonPlatforms.slice();
         }
+
         engines = cfg.getEngines(projectHome);
+
+        if(fs.existsSync(pkgJsonPath)) {
+            pkgJson = require(pkgJsonPath);
+        // TODO: Eventually refactor out to pacakge manager module.
```

Comment:
oo you can keep it. Would be good practice to include the issue number in the todo.